src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -61,6 +61,9 @@ catch (const wil::ResultException& exception)
         return E_PENDING;
     }
 
+#ifndef NDEBUG

```

Comment:
Leonard this may be controversial, but I think that the warning prompt is only useful for shader and font and etc problems the user may stand a chance of fixing

Reply:
...Perhaps we should instead check if the dialog is already being shown and then ignore a 2nd and following error/warning?

Reply:
Unfortunately, most of our few hundred million users interact with terminal because they have to, not because they want to.

How would you feel about keeping the dialog for preview/canary/dev and disabling it for stable?